### ISSUE TITLE
Confusing error message for `aqua dist deploy` when the specified service name is missing from the config 

### ISSUE THREAD

**Issue by kmd-fl** [fluencelabs/aqua#458]
When running the `aqua dist deploy --addr ... --data-path config.json --serivce not-in-config` command I get
```
scala.scalajs.js.JavaScriptException: TypeError: Cannot read properties of undefined (reading 'modules')
  at fillWithEmptyArrays(file:///usr/lib/node_modules/@fluencelabs/aqua/dist/config.js:2:29)
  at aqua.dist.DistOpts$.fillConfigOptionalFields(file:///usr/lib/node_modules/@fluencelabs/aqua/aqua.js:39226:82)
  at {anonymous}()(file:///usr/lib/node_modules/@fluencelabs/aqua/aqua.js:39395:32)
```

**Comment by DieMyst** [comment]
What was in config.json?

**Comment by DieMyst** [comment]
Fixed in #457
